### PR TITLE
Doctor has a confirmation page after submitting the report

### PR DESCRIPTION
**Pull request by dsamojlenko**
# What this does
Updates the confirmation screen when a doctor submits a report [according to the design described here](https://trello.com/c/77QgwYd6/539-doctor-has-a-confirmation-page-after-submitting-the-report).

NOTE: This does not include the new *Your information* or *Declaration/Review* screens

## To test
- Create a medical report
- Fill out all the sections
- Submit the report
- Notice the new Confirmation screen

### CODE REVIEW

**Comment by CalvinRodo** [comment]
Should we deactivate the code after the submission as the text on the page states?

**Review by CalvinRodo — DISMISSED** [review]
Is this just a design or should it have functionality changes as well?

**Comment by dsamojlenko** [comment]
I don't think we have a plan yet for how we're actually activating/deactivating codes. I was just doing the design portion here, but we do need to get to that...

**Comment by keithiopia** [comment]
> I don't think we have a plan yet for how we're actually activating/deactivating codes. I was just doing the design portion here, but we do need to get to that...

@CalvinRodo @dsamojlenko Current thinking is that the code is deactivated as soon as the MR is submitted, but the message saying it's already been submitted remains for 12 weeks: https://trello.com/c/MOaKInr8/498-noone-can-edit-the-report-after-it-has-been-submitted#comment-5e3443d6a4c12a1952bfd3e6

I'm not sure that part is in the scope of this work though.

File: config/locales/en.json [hunk 1]
```diff
@@ -188,11 +188,12 @@
...
 	"conditions.title": "Medical conditions",
-	"confirmation.intro": "Confirmation Intro goes here",
-	"confirmation.save_data": "Save data",
-	"confirmation.thank_you": "Thank you!",
-	"confirmation.thank_you_participating": "Thank you for participating!",
-	"confirmation.title": "Confirmation Title",
+	"confirmation.title": "You have submitted %s's medical report",
+	"confirmation.you_have_submitted": "You have submitted a medical report to support %s's application for Canada Pension Plan Disability Benefits",
+	"confirmation.what_happens_next": "What happens next",
+	"confirmation.what_happens_next_1": "Service Canada will begin assessing %s's application for Canada Pension Plan Disability Benefits. Service Canada might contact you in the future if they need any further information about %s's health or work capacity.",
```
Comment (keithiopia) [inline config/locales/en.json]: The second `%s` is rendering as 'undefined' for some reason.

File: config/locales/en.json [hunk 2]
```diff
@@ -188,11 +188,12 @@
...
+	"confirmation.what_happens_next_2": "The unique code you used is now disabled - you can no longer access or edit the medical report",
```
Comment (keithiopia) [inline config/locales/en.json]:
```suggestion
	"confirmation.what_happens_next_2": "The unique code you used is now disabled – you can no longer access or edit the medical report.",
```
Just a bit of a punctuation fix.

File: config/locales/en.json [hunk 3]
```diff
@@ -188,11 +188,12 @@
 	"conditions.save_condition": "Save condition",
 	"conditions.symptoms_began": "Symptoms began: ",
 	"conditions.title": "Medical conditions",
-	"confirmation.intro": "Confirmation Intro goes here",
-	"confirmation.save_data": "Save data",
-	"confirmation.thank_you": "Thank you!",
-	"confirmation.thank_you_participating": "Thank you for participating!",
-	"confirmation.title": "Confirmation Title",
+	"confirmation.title": "You have submitted %s's medical report",
+	"confirmation.you_have_submitted": "You have submitted a medical report to support %s's application for Canada Pension Plan Disability Benefits",
```
Comment (keithiopia) [inline config/locales/en.json]:
```suggestion
	"confirmation.you_have_submitted": "You have submitted a medical report to support %s's application for Canada Pension Plan Disability Benefits.",
```
Just adds a full stop/period to the end.

**Review by keithiopia — CHANGES_REQUESTED** [review]
I'm ignoring some of the validation errors on the medical report entry screens as i think they're out of scope for this. 
Also the 'download report' doesn't function, but I think that's not in scope for this ticket either.

File: config/locales/en.json (hunk 1 above)
Comment (dsamojlenko) [reply config/locales/en.json]: That's because I forgot to pass the name in twice! (done)

**Comment by dsamojlenko** [comment]
> I'm ignoring some of the validation errors on the medical report entry screens as i think they're out of scope for this.
> Also the 'download report' doesn't function, but I think that's not in scope for this ticket either.

correct and correct. we'll get to those.

**Review by keithiopia — DISMISSED** [review]
Good job!

**Review by jeffmaher — APPROVED** [review]
LGTM 🍕

**Pull request merged**